src/System.Collections.NonGeneric/src/System/Collections/Comparer.cs

```diff
@@ -34,6 +43,36 @@ public Comparer(CultureInfo culture)
             _compareInfo = culture.CompareInfo;
         }
 
+#if netstandard17
```

Comment:
What tests are you referring to?  This is the source for Comparer.  I've also been told that in the near future we're no longer going to be compiling for anything earlier than 1.7, at which point all of these ifs can be removed; I'd prefer to just remove them rather than having to recombine files at that point.